lib/invisible_captcha/controller_ext.rb

```diff
@@ -22,6 +31,14 @@ def on_spam_action(options = {})
       end
     end
 
+    def on_threshold_spam_action(options = {})
+      if action = options[:on_threshold_spam]
+        send(action)
+      else
+        flash[:error] = InvisibleCaptcha.threshold_error_message
```

Comment:
In this case (threshold not reached), we probably should respond by default with something like: add the flash message (as you do) and redirect to back?